.github/workflows/actions.yml

```diff
@@ -48,10 +48,10 @@ jobs:
         OS: [macOS-15]
         SDK: ["appletvsimulator", "iphonesimulator", "watchsimulator", "xrsimulator"]
     steps:
-    - uses: maxim-lobanov/setup-xcode@v1
+    - uses: maxim-lobanov/setup-xcode@v1 # https://github.com/maxim-lobanov/setup-xcode
       with:
-        xcode-version: '16.0.0'
-    - uses: actions/checkout@v4
+        xcode-version: latest-stable
```

Comment:
We could hardcode it to '16.3.0' too. But in general:
- We have to update the DEVICE at least once a year.
- We have to update the stable Xcode at least 5 times a year.

To minimise the number of commits, we'll use `latest-stable` and only care about updating the DEVICE.